test/integration/sequelize.test.js

```diff
@@ -448,8 +450,10 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
             ].some(fragment => err.message.includes(fragment)));
           } else if (dialect === 'mssql') {
             expect(err.message).to.equal('Login failed for user \'bar\'.');
+          } else if (dialect === 'db2') {
+            expect(err.message).to.include('SQLSTATE=08004');
```

Comment:
anything more digestible?

Reply:
@AWADHAMBIKA check the full message by printing err.message and put the text here.

Reply:
Under review and will be resolved eventually